apps/app_rpt/rpt_manager.c

```diff
@@ -18,6 +18,22 @@
 
 extern struct rpt rpt_vars[MAXRPTS];
 
+static char *
+ctime_no_newline(const time_t *clock)
```

Comment:
static char* should be on the same line as the function name, with no space between char and *

Reply:
Fixed